apps/web/src/components/MenuBar.tsx

```diff
@@ -0,0 +1,31 @@
+import Link from 'next/link';
+import { useRouter } from 'next/router';
+
+import { menuBarTabs } from '@services';
+
+export const MenuBar: React.FC = () => {
+  const router = useRouter();
+
+  const active = `font-bold`;
```

Comment:
why do we need variable?

Reply:
I used it mainly for readability, to show the purpose of the class being shown